eq-author-api/README.md

```diff
@@ -49,110 +39,34 @@ Docker is installed for your platform, navigate to the project directory, then r
 
 Build the docker image (1st time run):
 
-```
-docker-compose build
-```
-
-```
-docker-compose up
-```
+    docker-compose build
 
-Once the containers are running you should be able to navigate to http://localhost:4000/graphiql and begin exploring the eQ Author GraphQL API.
+    docker-compose up
```

Comment:
I think it's preferable to wrap commands in code blocks.

e.g.

```
docker-compose up
```

Reply:
This has been resolved now.